Hospital.Tests/Tests.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hospital.Domain.Entities;
+using Xunit;
+
+namespace Hospital.Tests;
+
+/// <summary>
+/// Container for query methods and their unit tests.
+/// </summary>
+public class Tests
+{
+    /// <summary>
+    /// Provides LINQ queries for doctors, patients, and appointments.
+    /// </summary>
+    public static class Queries
+    {
+        /// <summary>
+        /// Returns all doctors with at least 10 years of experience.
+        /// </summary>
+        /// <param name="doctors">List of doctors to search.</param>
+        /// <returns>List of doctors meeting the criteria.</returns>
+        public static List<Doctor> DoctorsWith10Plus(List<Doctor> doctors) =>
+            doctors.Where(d => d.ExperienceYears >= 10).ToList();
```

Comment:
Collection expression:
```suggestion
            [.. doctors.Where(d => d.ExperienceYears >= 10)];
```